users/src/api/routes/users.routes.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const { filter } = require('../../util');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ce-box/CE5508-astraoke-api/pullRequest?prid=8478842)
```suggestion
const { filter } = require("../../util");
```